bag/datasets/bag/serializers.py

```diff
@@ -630,6 +630,16 @@ class NummeraanduidingDetail(BagMixin, rest.HALSerializer):
     grootstedelijkgebied = Grootstedelijkgebied()
     stadsdeel = Stadsdeel()
     openbare_ruimte = OpenbareRuimte()
+    verblijfsobject = serializers.HyperlinkedRelatedField(allow_null=True,
+        queryset=models.Verblijfsobject.objects.all(), required=False, view_name='verblijfsobject-detail',
+        lookup_field='landelijk_id', lookup_url_kwarg='pk')
+    ligplaats = serializers.HyperlinkedRelatedField(allow_null=True,
+        queryset=models.Ligplaats.objects.all(), required=False, view_name='ligplaats-detail',
+        lookup_field='landelijk_id', lookup_url_kwarg='pk')
+    standplaats = serializers.HyperlinkedRelatedField(allow_null=True,
+       queryset=models.Standplaats.objects.all(), required=False, view_name='standplaats-detail',
```

Comment:
Andere indent?

Reply:
Indent aangepast, hoewel ik dit eigenlijk teveel naar rechts vind. Maar het is inderdaad wel PEP8